findphotodates.py

```
@@ -0,0 +1,315 @@
+            
+    if len(date_counter) <= 20:
+        print(f"\n{file_type_label} Taken per day:")
+        for day, count in sorted(date_counter.items()):
+            print(f"  {day}: {count}")
+    else:
+        month_counter = defaultdict(int)
+        for day in date_counter:
+            try:
+                year_month = "-".join(day.split("-")[:2])  # Extract year and month
+                month_counter[year_month] += date_counter[day]
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)